jdaviz/conftest.py

```diff
@@ -241,14 +241,23 @@ def spectrum1d_cube_larger():
 
 @pytest.fixture
 def spectrum1d_cube_largest():
+    return _spectrum1d_cube_largest_custom_unit(fluxunit=u.Jy)
+
+
+@pytest.fixture
+def spectrum1d_cube_largest_custom_unit():
+    return _spectrum1d_cube_largest_custom_unit
```

Comment:
Missing function call here?

Reply:
This is intentional. It let's us use the same function for two fixtures:

https://github.com/spacetelescope/jdaviz/blob/5c3855728fb03b752dfba3678813373a4b982a04/jdaviz/conftest.py#L242-L260